src/Razor/src/Microsoft.AspNetCore.Razor.VSCode.Extension/package.json

```diff
@@ -78,6 +78,11 @@
                     "description": "The URL of the application",
                     "default": "https://localhost:5001"
                   },
+                  "debugProxyVersion": {
```

Comment:
We need to know what version of the debug proxy package we should install on the users host machine. We can do some cleverness here to read the version of the DevServer mentioned in the user's project file. I opted for an explicit option here to avoid having to write csproj-parsing code. Users will need to provide the debugging proxy version to use.

Coincidentally, this will also allow people to pick up bug fixes in the debugging proxy without having to update their entire Blazor project.